src/ampal/protein.py

```diff
@@ -555,19 +555,26 @@ def n_join(self, other, psi=-40.76, omega=-178.25, phi=-65.07,
         self.tags['assigned_ff'] = False
         return
 
-    def tag_sidechain_dihedrals(self, force=False):
+    def tag_sidechain_dihedrals(self, force=False, tag_rotamers=False):
```

Comment:
Thinking about this, maybe we should always tag the rotamers with their Dunbrack labels, it would simplify the whole thing.

Reply:
I agree